packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/mod.rs

```diff
@@ -4,14 +4,12 @@ mod link_type;
 mod property_type;
 mod read;
 
-use type_system::{uri::VersionedUri, DataType, PropertyType};
-
-use crate::ontology::types::{EntityType, LinkType};
+use type_system::{uri::VersionedUri, DataType, EntityType, LinkType, PropertyType};
 
 /// Provides an abstraction over elements of the Type System stored in the Database.
 ///
 /// [`PostgresDatabase`]: crate::store::PostgresDatabase
-pub trait OntologyDatabaseType {
+pub trait OntologyDatabaseType: Into<serde_json::Value> + TryFrom<serde_json::Value> {
```

Comment:
Usually, I'm not a fan of putting bounds on the traits (or structs) unless required (e.g. for associated types for `Drop`). In how many places does this need changes when moving the bounds to the implementation?

Reply:
I have updated it to remove the bounds :)